app-connector-client/server/services.js

```diff
@@ -8,38 +8,55 @@ const connection = require("./connection")
 const OAUTH = "/authorizationserver/oauth/token"
 const METADATA = "/metadata"
 const request = require("request-promise")
+var success_count = 0;
+var failed_count = 0;
+var apis_success = [];
+var apis_failed = [];
 module.exports = {
     createServicesFromConfig: createServicesFromConfig,
     getAllAPI: getAllAPI,
     getAllAPIs, getAllAPIs,
     createAPI: createAPI,
     updateAPI: updateAPI,
-    fillServiceMetadata: fillServiceMetadata
+    fillServiceMetadata: fillServiceMetadata,
+    getStatus: getStatus,
+    createEventsFromConfig: createEventsFromConfig,
+    fillEventData: fillEventData
 }
 
 async function createServicesFromConfig(hostname, apisConfig, registeredApis) {
     if (!apisConfig)
         return
 
     var error_message = ""
+    success_count = 0;
+    failed_count = 0;
+    apis_success = [];
+    apis_failed = [];
     for (var i = 0; i < apisConfig.length; i++) {
         var api = apisConfig[i]
         try {
             var reg_api
             if (registeredApis.length > 0)
                 reg_api = registeredApis.find(x => x.name == api.name)
             if (!reg_api) {
-                await createService(api, hostname)
+                await createService(api, false, hostname)
                 LOGGER.debug("Registered API successful: %s", api.name)
             }
             else {
-                await updateService(api, reg_api.id, hostname)
+                await updateService(api, reg_api.id, false, hostname)
                 LOGGER.debug("Updated API successful: %s", api.name)
             }
+            success_count++;
+            apis_success.push(api.name);

```

Comment:
success_count == apis_success.length, no need to have two variables?

Reply:
that's true